src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -929,8 +929,12 @@ internal static Cci.IGenericParameterReference Translate(TypeParameterSymbol par
             DiagnosticBag diagnostics,
             bool needDeclaration = false)
         {
+            if (fieldSymbol.IsTupleField)
```

Comment:
Sure, I'll change

---
In reply to: [61800802](https://github.com/dotnet/roslyn/pull/11006#discussion_r61800802) [](ancestors = 61800802)